Makefile

```diff
@@ -1038,3 +1038,24 @@ dev-lab-kwok: ## Deploy kwok
 .PHONY: help
 help: ## Shows the available commands
 	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-40s\033[0m %s\n", $$1, $$2}'
+
+#####################
+# Bump new versions #
+#####################
+
+KYVERNO_CHART_VERSION ?= 3.1.0-rc.7
+POLICIES_CHART_VERSION ?= 3.1.0-rc.7
+APP_VERSION ?= v1.11.0-rc.7
```

Comment:
Let's not change it on main.

Reply:
can you please elaborate what you exactly want to say